charts/rancher-monitoring/103.0.3+up45.31.1/values.yaml

```diff
@@ -560,7 +560,7 @@ upgrade:
   ## Image to use to run the scripts
   image:
     repository: rancher/shell
-    tag: v0.1.22
```

Comment:
this also might need to be reverted: https://github.com/rancher/charts/commit/eac7433f2444e85e09ecae00103d57bcbb2b6b1a